static/app/views/performance/browser/resources/jsCssView/resourceTable.tsx

```diff
@@ -56,18 +57,18 @@ function ResourceTable({sort}: Props) {
   });
 
   const columnOrder: GridColumnOrder<keyof Row>[] = [
-    {key: SPAN_DESCRIPTION, width: COL_WIDTH_UNDEFINED, name: 'Resource name'},
-    {key: SPAN_OP, width: COL_WIDTH_UNDEFINED, name: 'Type'},
-    {key: `avg(${SPAN_SELF_TIME})`, width: COL_WIDTH_UNDEFINED, name: 'Avg Duration'},
+    {key: SPAN_DESCRIPTION, width: COL_WIDTH_UNDEFINED, name: t('Resource Description')},
```

Comment:
let's just use name instead of description.

we're gonna slowly phase out description anyway with the new spans schema (aligns with otel, transaction name).

Reply:
I like name more, but I thought it would make sense to say description in order to be consistent with the queries module

Reply:
tbh we should just change it in queries as well then.

this review doesn't block merge, feel free to go for it.